src/main/java/org/retest/rebazer/domain/BitbucketContent.java

```diff
@@ -0,0 +1,11 @@
+package org.retest.rebazer.domain;
+
+import lombok.Builder;
+import lombok.Value;
+
+@Value
+@Builder
```

Comment:
The class is rather simple, is there a reason you have chosen `@Builder`? Why the class in the first place, isn't it just a wrapper for a string or is required somewhere else?

Reply:
I needed it to get another nesting depth, however there was a simpler solution dismiss at this point